Maple2.Server.World/Service/GlobalService.cs

```diff
@@ -55,14 +69,20 @@ public override Task<LoginResponse> Login(LoginRequest request, ServerCallContex
             });
         }
 
-        // TODO: Implement password check
+        bool isPasswordValid = BCrypt.Net.BCrypt.Verify(password, account.Password);
+        if (!isPasswordValid) {
+            return Task.FromResult(new LoginResponse {
+                Code = LoginResponse.Types.Code.ErrorPassword,
+                Message = "Incorrect Password."
+            });
+        }
 
         if (account.MachineId != machineId) {
-            logger.Warning("MachineId mismatch for account {AccountId}", account.Id);
+            logger.Warning($"MachineId mismatch for account {account.Id}");
```

Comment:
for logging its not recommended using interpolation

Reply:
done